### DISCUSSION TITLE
uscita giustificata

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#366] [Idee e proposte]
Buongiorno,sarebbe allora possibile implementare la giustifica anche per l'uscita anticipata?E' ormai da due anni che facciamo la stessa richiesta spiegando tutte le motivazioni

**Comment by trinko** [comment]
Leggi:
[ Gestione ritardi e uscite anticipate](https://github.com/iisgiua/giuaschool/discussions/376)
